Makefile

```diff
@@ -22,14 +25,14 @@ doc: dg
 	mkdir -p ./root/
 	${GOMD2MAN} -in=help/help.md.rendered -out=./root/help.1
 
-build: doc dg
+build: dependencies doc dg
```

Comment:
I don't like this. Every time I'll do a local `make build` dnf will be invoked and will fail if not ran as root. Two options:

1. call `make dependencies` explicitly
2. create a new target which will do exactly this

Reply:
@TomasTomecek It make sense. In case there will be general agreement, that there will exist everytime requirement.sh script, it will fit exactly what I want. and in case it will be missing means no special requirements for build and installation.